deps/rabbit/test/logging_SUITE.erl

```diff
@@ -1383,7 +1383,9 @@ log_and_return_json_object(Context, Metadata, DecodeOpts) ->
                         Content,
                         "^.+\"" ++ RandomMsg ++ "\".+$",
                         ReOpts),
-    Term = rabbit_json:decode(Line, [{labels, attempt_atom} | DecodeOpts]),
+
+    Term0 = rabbit_json:decode(Line, DecodeOpts),
```

Comment:
It looks like these `DecodeOpts` may need to be adjusted to what thoas expects. I see `[return_maps]` being passed to `log_and_return_json_object/3` elsewhere in this suite

Along these lines I see some options being passed in `deps/rabbitmq_ct_helpers/src/rabbit_mgmt_test_util.erl:50` and `deps/rabbitmq_aws/src/rabbit_aws_json.erl:17` that are in jsx format (specifically `[{return_maps, false}]`)

Reply:
Hm, yeah, there is no alternative in `thoas` so functions such as `http_get_as_proplist/2` would have to convert the returned value themselves. Let me see where that function is used.